packages/delegation-process/src/model/domain/apiConverter.ts

```diff
@@ -0,0 +1,96 @@
+import { delegationApi } from "pagopa-interop-api-clients";
+import {
+  Delegation,
+  DelegationContractDocument,
+  DelegationKind,
+  DelegationStamp,
+  DelegationStamps,
+  DelegationState,
+} from "pagopa-interop-models";
+import { match } from "ts-pattern";
+
+export const delegationToApiDelegation = (
+  delegation: Delegation
+): delegationApi.Delegation => ({
+  id: delegation.id,
+  delegatorId: delegation.delegatorId,
+  delegateId: delegation.delegateId,
+  eserviceId: delegation.eserviceId,
+  createdAt: delegation.createdAt.toJSON(),
+  submittedAt: delegation.submittedAt.toJSON(),
+  approvedAt: delegation.approvedAt
+    ? delegation.approvedAt.toJSON()
+    : undefined,
+  rejectedAt: delegation.rejectedAt
+    ? delegation.rejectedAt.toJSON()
+    : undefined,
+  rejectionReason: delegation.rejectionReason || undefined,
+  revokedAt: delegation.revokedAt ? delegation.revokedAt.toJSON() : undefined,
+  state: delegationStateToApiDelegationState(delegation.state),
+  kind: delegationKindToApiDelegationKind(delegation.kind),
+  contract: delegation.contract
+    ? delegationContractToApiDelegationContract(delegation.contract)
+    : undefined,
+  stamps: delegationStampsToApiDelegationStamps(delegation.stamps),
+});
+
+export const delegationStateToApiDelegationState = (
+  state: DelegationState
+): delegationApi.DelegationState =>
+  match<DelegationState, delegationApi.DelegationState>(state)
+    .with("Active", () => "ACTIVE")
```

Comment:
Same as the other implementations, we use the object enum imported from the models here

```suggestion
    .with(delegationState.active, () => "ACTIVE")
```